blocks/components/editable/index.js

```diff
@@ -2,11 +2,13 @@
  * External dependencies
  */
 import classnames from 'classnames';
-import { last, isEqual, capitalize, omitBy, forEach, merge } from 'lodash';
+import { last, isEqual, capitalize, omitBy, forEach, merge, drop, pick, compact } from 'lodash';
 import { nodeListToReact } from 'dom-react';
 import { Fill } from 'react-slot-fill';
 import 'element-closest';
 
+const { tinymce } = window;
```

Comment:
I'm thinking we should change to treat `tinymce` as a proper module, even if it's just a reference to the window object configured through Webpack. Dunno that we need to make this change here, since we're already assuming the global in `tinymce.js` as well.